content/user/template/template_switching.md

```diff
@@ -9,15 +9,27 @@ The process of changing templates can be done using the [Template Selection](/us
 
 But what if the templates you are switching between expect different configuration values?  It's tedious to do these by hand, especially if you're switching back and forth doing tests.
 
-That's why Zen Cart 1.5.8a introduces a new feature called `template_init`.  Place a file called `template_init.php` in the includes/templates/YOURTEMPLATE folder.  It will run when the template  is set to YOURTEMPLATE.
+That's why Zen Cart 1.5.8a introduces a new feature called `template_init`.  
+Place a file called `template_init.php` in the includes/templates/YOURTEMPLATE folder.  
+It will execute when the template is set to YOURTEMPLATE.  
+Note this functionality is not available/these queries are not executed when using  [Private Template Testing](/user/template/private_testing/) .
 
 For example, here's a template init that turns off CSS buttons when this template is selected: 
 
 ```
 <?php
 // Settings for Older Template
-$db->Execute("UPDATE " . TABLE_CONFIGURATION . " SET configuration_value='No' WHERE configuration_key  = 'IMAGE_USE_CSS_BUTTONS'");
+$db->Execute('UPDATE ' . TABLE_CONFIGURATION . ' SET configuration_value="No" WHERE configuration_key = "IMAGE_USE_CSS_BUTTONS"');
```

Comment:
Please undo this.  You'll see that the standard for $db->Execute is double quotes throughout the code base.

Reply:
I see a mixture of styles. 
From what I have read, they should always be single quotes unless there is a reason to be otherwise, as the contents of double quotes get evaluated, so in most cases, for nothing/wasted processing.

Reply:
Right.  Please do what I'm asking so I can merge this.